docs/XDCR.md

```diff
@@ -0,0 +1,56 @@
+# XDCR Protostellar Behaviour
+
+### PushDocument
+
+PushDocument exposes two values for CAS, one is the CheckCas and one is the StoreCAS. The
+CheckCas represents the typical Compare And Swap operation of other memcached commands. If
+a CheckCas is specified, conflict resolution is disabled and assuming the CAS check completes,
+the write will be accepted. If CheckCas is set to the value of '0' explicitly, it is expected
```

Comment:
>  If a CheckCas is specified, conflict resolution is disabled and assuming the CAS check completes,
> the write will be accepted.

I think I'm misunderstanding this. I thought that performing the CAS check was conflict resolution? Is this supposed to say: "If CheckCas is specified, conflict resolution is enabled..."?

Reply:
The logic here is that if the caller already has the original document, it implies that they can perform whatever kind of conflict resolution they would like.